client/my-sites/stats/stats-subscribers-page/index.jsx

```diff
@@ -0,0 +1,97 @@
+import JetpackColophon from 'calypso/components/jetpack-colophon';
+import Main from 'calypso/components/main';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import { getSuggestionsVendor } from 'calypso/lib/domains/suggestions';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import AnnualHighlightsSection from '../annual-highlights-section';
+import Followers from '../stats-followers';
+import Reach from '../stats-reach';
+
+const StatsSubscribersPage = ( props ) => {
+	const { siteId, siteSlug, translate, isOdysseyStats, isJetpack } = props;
+
+	const isSubscribersPageEnabled = config.isEnabled( 'stats/subscribers-section' );
+
+	const statsModuleListClass = classNames(
+		'stats__module-list stats__module--unified',
+		{
+			'is-odyssey-stats': isOdysseyStats,
+			'is-jetpack': isJetpack,
+		},
+		'is-insights-page-enabled'
+	);
+
+	// Track the last viewed tab.
+	// Necessary to properly configure the fixed navigation headers.
+	// sessionStorage.setItem( 'jp-stats-last-tab', 'subscribers' );
+
+	// TODO: should be refactored into separate components
+	/* eslint-disable wpcalypso/jsx-classname-namespace */
+	return (
+		<Main fullWidthLayout>
+			<DocumentHead title={ translate( 'Jetpack Stats' ) } />
+			<PageViewTracker path="/stats/subscribers/:site" title="Stats > Subscribers" />
+			<div className="stats">
+				<FormattedHeader
+					brandFont
+					className="stats__section-header modernized-header"
+					headerText={ translate( 'Jetpack Stats' ) }
+					subHeaderText={ translate( "View your site's performance and learn from trends." ) }
+					align="left"
+				/>
+				<StatsNavigation selectedItem="subscribers" siteId={ siteId } slug={ siteSlug } />
+				{ isSubscribersPageEnabled && (
+					<>
+						{ /* TODO: replace annual highlight */ }
+						<AnnualHighlightsSection siteId={ siteId } />
+						{ siteId && (
+							<DomainTip
+								siteId={ siteId }
+								event="stats_subscribers_domain"
+								vendor={ getSuggestionsVendor() }
+							/>
+						) }
+						{ config.isEnabled( 'stats/subscribers-section' ) && (
+							<AsyncLoad require="calypso/my-sites/stats/subscribers-section" siteId={ siteId } />
+						) }
+						<div className={ statsModuleListClass }>
+							<Followers path="followers" />
+							<Reach />
+						</div>
+					</>
+				) }
+				<JetpackColophon />
+			</div>
+		</Main>
+	);
+	/* eslint-enable wpcalypso/jsx-classname-namespace */
+};
+
+StatsSubscribersPage.propTypes = {
+	translate: PropTypes.func,
+};
+
+const connectComponent = connect( ( state ) => {
+	const siteId = getSelectedSiteId( state );
+	const isOdysseyStats = config.isEnabled( 'is_running_in_jetpack_site' );
+	return {
+		siteId,
+		siteSlug: getSelectedSiteSlug( state, siteId ),
+		isOdysseyStats,
+		isJetpack: isJetpackSite( state, siteId ),
+	};
+} );
```

Comment:
In a follow-up PR: We could use [`useSelector`](https://react-redux.js.org/api/hooks#useselector) instead; this will allow us to be more consistent with our hook usage.

Reply:
Tracked in #75378.